src/modules/tabs/tabs.tsx

```diff
@@ -59,6 +68,17 @@ export const Tabs: React.FC<IProps> = ({
       }}
       keyExtractor={(item, index) => index.toString()}
     />
+
+    {linkLabel && (
+      <StyledLink
+        testID="link"
+        activeOpacity={0.75}
+        onPress={() => onLinkPress && onLinkPress()}>
```

Comment:
Can we please remove this check?

Reply:
Actually this check will require here. because I have defined onLinkPress as an optional prop. So, if I remove this prop then It will throw undefined error here.

Reply:
okay

Reply:
On a second thought, if we pass a `linkLabel` then the link will show up on the UI and if nothing is passed as the onPress, then it would simply do nothing. Now if we have a link that is visible and not doing anything on press, that's not a good thing.

> So, if I remove this prop then It will throw undefined error here.

I would say let the app throw an error because it's a mistake that the developer is making. So let the developer know that they forgot to pass the `onLinkPress` prop.

Reply:
> if we pass a linkLabel then the link will show up on the UI and if nothing is passed as the onPress, then it would simply do nothing. Now if we have a link that is visible and not doing anything on press, that's not a good thing.

I agree with you but then in this case, I need to make onLinkPress a mandatory prop. and if there is a case when developer don't want to use link labels in tab component then it will turn out to be an inappropriate way of passing onLinkPress strictly. Also, for eg., when we implement any Touchable then it also does not ask for an onPress prop to be passed strictly. I hope you understand my point.

Reply:
I get your point but i'd say this, please make the prop as optional but let us not do the check. We don't want to allow touchable's that don't do anything. And even if that is required (in some weird case), let the consumer take care of whether the link should do something or nothing.

> when we implement any Touchable then it also does not ask for an onPress prop to be passed strictly

Exactly, but what good is a touchable if it doesn't do anything on touch/press? Let us remove the check please.

Thanks!

Reply:
Okay Ajay, I am removing this check. So now the behaviour would be: consumer will required to pass onLinkPress along with linkLabel prop or else it will throw an error if link label is pressed.
Thanks.